sys/include/shell.h

```diff
@@ -75,6 +75,15 @@ extern "C" {
 
 /**
  * @brief Default shell buffer size (maximum line length shell can handle)
+ *
+ * @warning On terminals that buffer input and send the full command line in one go,
```

Comment:
```suggestion
 * @warning On terminals that buffer input and send the full command line in one go (like `pyterm`),
```

let's not be vague, this affects our default terminal after all

Reply:
To be frank I don't have the overview of which terminal program we default to, we support so many and I often just socat in to the UART.

If it is really the default for all, we could even say

```suggestion
 * @warning On terminals that buffer input and send the full command line in one go (like our default `pyterm`),
```

but then I'd also look into sharpening the criterion, as it does apparently not affect all stdio. (I guess that UART based could be unaffected because UARTs' units of transfer are smaller than the buffer size, and slow enough that the shell thread can catch up).

Reply:
New text now says "When terminals that buffer input and send the full command line in one go are used on stdin implementations with fast bursts of data", and

```
+ *   For example, this affects systems with direct USB stdio (@ref
+ *   usbus_cdc_acm_stdio) with the default terminal `pyterm`.
```

(Sorry, should have been a fixup rather than an amend/force)